apps/dashboard/src/components/workflow-editor/steps/skip-conditions-button.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button } from '@/components/primitives/button';
+import { Separator } from '@/components/primitives/separator';
+import { SidebarContent } from '@/components/side-navigation/sidebar';
+import { useFeatureFlag } from '@/hooks/use-feature-flag';
+import { FeatureFlagsKeysEnum, StepResponseDto } from '@novu/shared';
+import { useMemo } from 'react';
+import { RiArrowRightSLine, RiGuideFill } from 'react-icons/ri';
+import { RQBJsonLogic } from 'react-querybuilder';
+import { parseJsonLogic } from 'react-querybuilder/parseJsonLogic';
+import { Link } from 'react-router-dom';
+
+export function SkipConditionsButton({ step, inSidebar = false }: { step: StepResponseDto; inSidebar?: boolean }) {
+  const isStepConditionsEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_STEP_CONDITIONS_ENABLED);
+  const uiSchema = step.controls.uiSchema;
+  const skip = uiSchema?.properties?.skip;
+
+  const conditionsCount = useMemo(() => {
+    if (!step.controls.values.skip) return 0;
+
+    const query = parseJsonLogic(step.controls.values.skip as RQBJsonLogic);
+
+    return query.rules.length;
+  }, [step]);
+
+  const button = (
```

Comment:
if we need to reuse this button, please extract it to a separate function so we can render it by <Button/> instead.

Reply:
Replied here: https://github.com/novuhq/novu/pull/7592/files/6c796518c845186a2b8445177502bddf8e58a628#r1930815362

Reply:
let's not complicate this for now, as it will always be in the sidebar